code/datums/uplink_items/uplink_traitor.dm

```diff
@@ -41,6 +41,15 @@
 	item = /obj/item/storage/box/syndie_kit/fake_revolver
 	cost = 1
 	job = list("Clown")
+
+/datum/uplink_item/jobspecific/trick_grenade
+	name = "Trick Grenade"
+	desc = "Syndicate Minibomb with glue ejectors that will stick it to the user's hands on activation."
+	reference = "CGN"
+	item = /obj/item/storage/box/syndie_kit/fake_minibomb
+	cost = 1
```

Comment:
```suggestion
	cost = 5
```
This will need to be updated to account for the new TC Inflation PR that was just merged.